### PR TITLE
fix: correct legend inconsistencies

### PR DESCRIPTION
**Pull request by cthulhustig**

### CODE REVIEW

**Comment by inexorabletash** [comment]
The SVGs should be generated by loading legend.html (somewhere in res/) in a browser and saving the embedded results. If those don't match up something is very wrong somewhere!

Did you use that approach or update them manually?

**Comment by inexorabletash** [comment]
Ah, it appears the Legend sector actually has the labels hardcoded in white, which isn't helpful. 

https://github.com/inexorabletash/travellermap/blob/main/res/Sectors/Meta/legend.xml

Hmmm...

File: index.css [hunk 1]
```diff
@@ -833,25 +833,38 @@ body:not(.route-shown) #routeBox { min-height: 135px; max-height: 135px; }
 
 #legendBox .glyph { display: inline-block; width: 20px; text-align: center; }
 #legendBox .highlight { color: #E32736; }
-.style-draft #legendBox .highlight,
```
[the remark this answers is not in the document]
Comment (inexorabletash) [reply index.css:836]: > draft doesn't use the highlight colour for capitals like other styles, instead it appears to uses it to highlight the name of low populations worlds

That doesn't match what I'm seeing or the code. Can you give an example?

image

File: res/ui/Legend_1003_draft.svg
Comment (inexorabletash) [inline res/ui/Legend_1003_draft.svg:1]: I updated this file separately, but it differs by the newline so it's not showing as a no-op. Can you revert the change to this file?

File: res/ui/Legend_1006_draft.svg
Comment (inexorabletash) [inline res/ui/Legend_1006_draft.svg:1]: I updated this file separately, but it differs by the newline so it's not showing as a no-op. Can you revert the change to this file?

File: index.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (inexorabletash) [reply index.css:836]: The change looks good other than this file. If you revert this line change I can merge it.

**Review by inexorabletash — CHANGES_REQUESTED** [review]

File: index.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (cthulhustig) [reply index.css:836]: Sorry I think I must have gotten myself a little confused. You're correct, draft does use the red highlight colour for drawing capital names on the map. So I think that means my change to stop the legend using grey as the highlight colour is good as it means the legened shows bases in red and the capital highlight string in red.

While trying to work out how I got it so wrong I also noticed that draft always uses upper case for world names and underlines the names of high pop worlds so I've pushed a change to make that update.

File: res/ui/Legend_1003_draft.svg
Comment (cthulhustig) [reply res/ui/Legend_1003_draft.svg:1]: I'm not sure what I need to do for this, I'm still learning some of the intricacies of git. Now that you've merged main into my branch won't it be the versions you regenerated that end up on main rather than version that was part of my initial commit?

**Comment by cthulhustig** [comment]
> The SVGs should be generated by loading /res/ui/legend.html in a browser and saving the embedded results. If those don't match up something is very wrong somewhere!
> 
> Did you use that approach or update them manually?

I didn't realise that was how they were generated so I had just edited the svg manually

File: res/ui/Legend_1003_draft.svg
Comment (cthulhustig) [reply res/ui/Legend_1003_draft.svg:1]: Could the newline issue be because .gitattributes doesn't have an entry for .svg?

**Comment by inexorabletash** [comment]
Thanks!
